### PR TITLE
添加docker-compose，优化dockerfile

### PR DESCRIPTION
**Pull request by hryGithub**

### CODE REVIEW

**Comment by hryGithub** [comment]
修改 commit email——address

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build and publish images
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: hyr326/apioak
```
Comment (shuaijinchao) [inline .github/workflows/dockerimage.yml]: this name can be changed to apioak/apioak.

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+# apioak-docker
+    docker run -d -p 10080:10080 -e DB_HOST= -e DB_PORT= -e DB_NAME= -e DB_USER= -e DB_PASSWORD=  hyr326/apioak:latest
+
+# docker-compose
+    git clone https://github.com/apioak/docker.git
+    cd docker
+    docker-compose up 
+
+# build 
+    git clone https://github.com/apioak/docker.git
```
Comment (shuaijinchao) [inline README.md]: the warehouse name can be modified to apioak/apioak-docker

**Review by shuaijinchao — APPROVED** [review]

**Comment by shuaijinchao** [comment]
thanks, @hryGithub

**Pull request merged**